.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,83 @@
+name: guillotina
+
+on: [push]
+
+jobs:
+
+  # Job to run pre-checks
+  pre-checks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install package
+        run: |
+          pip install $(grep "Cython" requirements.txt)
+          pip install -r requirements.txt
+          pip install -r contrib-requirements.txt
+          pip install -e .[test]
+          pip install -e .[testdata]
+      - name: Run pre-checks
+        run: |
+          flake8 guillotina --config=setup.cfg
+          mypy guillotina/ --ignore-missing-imports
+          isort -c -rc guillotina/
+          black --check --verbose guillotina
+  # Job to run tests
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
+        database: ["DUMMY", "postgres", "cockroachdb"]
+        db_schema: ["custom", "public"]
+
+    # Set environment variables
+    env:
+      DATABASE: ${{ matrix.database }}
+      DB_SCHEMA: ${{ matrix.db_schema }}
+
+    steps:
+      - name: Check supported matrix combinations
```

Comment:
What does this check ? Looks like we only check on PG and public ???

Reply:
We don't support the following combinations of the matrix:
DATABASE="cockroach", DB_SCHEMA="custom"
DATABASE="DUMMY", DB_SCHEMA="custom"

So that was the only way I found to skip running tests on those.

Reply:
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-excluding-configurations-from-a-matrix

Reply:
oh nice, that must be new, because I didn't see it when I was doing it! thanks @bloodbare